ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -99,7 +100,11 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       await profileApi().updateProfile(this.state.profileEdits);
       await reload();
     } catch (e) {
-      console.error(e);
+      if (e instanceof Response) {
```

Comment:
Similarly, can we use https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/retry.tsx#L34?